xhydro/testing/utils.py

```diff
@@ -0,0 +1,80 @@
+"""Utilities for testing and releasing xhydro."""
+
+import os
+import re
+from io import StringIO
+from pathlib import Path
+from typing import Optional, TextIO, Union
+
+
+def publish_release_notes(
+    style: str = "md", file: Optional[Union[os.PathLike, StringIO, TextIO]] = None
+) -> Optional[str]:
+    """Format release history in Markdown or ReStructuredText.
+
+    Parameters
+    ----------
+    style: {"rst", "md"}
+      Use ReStructuredText formatting or Markdown. Default: Markdown.
+    file: {os.PathLike, StringIO, TextIO}, optional
+      If provided, prints to the given file-like object. Otherwise, returns a string.
```

Comment:
```suggestion
    style : {"rst", "md"}
        Use ReStructuredText (`rst`) or Markdown (`md`) formatting. Default: Markdown.
    file : {os.PathLike, StringIO, TextIO, None}
        If provided, prints to the given file-like object. Otherwise, returns a string.
```